kafka-connect-s3/pom.xml

```diff
@@ -73,21 +73,6 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
-            <exclusions>
-            <exclusion>
```

Comment:
@janjwerner-confluent - does bumping AWS version to 1.12.524 solve for this exclusion ( CVE-2020-28491 )?

Reply:
yes, please see: https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-core/1.12.524
if you check https://twistlock.tools.confluent-internal.io/#!/monitor/vulnerabilities/images/ci?search=Confluent%20Public%20Repo%20PR%20builder%2Fkafka-connect-storage-cloud%2FPR-660
the version brought in is: 2.13.4 - /tmp/repo/kafka-connect-s3/target/kafka-connect-s3-10.0.22-SNAPSHOT-package/share/java/kafka-connect-s3/jackson-dataformat-cbor-2.13.4.jar

Reply:
ok.